### PR TITLE
[GCU] [MA] Adding support in existing tests - mgmt interface

### PR DESCRIPTION
**Pull request by okaravasi**
### Description of PR

This PR adds changes for adding Multi-ASIC support for generic config updater (GCU) in existing test_mgmt_interface suite.
Original ticket that got splitted: https://github.com/sonic-net/sonic-mgmt/pull/14070
This change has a dependency on common changes added via https://github.com/sonic-net/sonic-mgmt/pull/15182

Summary:
Fixes # (issue)

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [X ] Test case(new/improvement)

### Back port request
- [ ] 202012
- [ ] 202205
- [ ] 202305
- [ ] 202311
- [ X] 202405

#### What is the motivation for this PR?
To add  support for Multi-ASIC testing in existing GCU suite generic_config_updater/test_mgmt_interface.py.
#### How did you verify/test it?
Ran in vs-kvm-t0 testbed. Ran in t2 MA platform.

### CODE REVIEW

**Comment by okaravasi** [comment]
@xincunli-sonic  Could you please review?

**Comment by xincunli-sonic** [comment]
Hi @okaravasi , could you resync the code base from master, seems like you missed one change: https://github.com/sonic-net/sonic-mgmt/pull/14098. Once you updated, I will start review. Thanks!

**Comment by okaravasi** [comment]
> Hi @okaravasi , could you resync the code base from master, seems like you missed one change: #14098. Once you updated, I will start review. Thanks!

Hi @xincunli-sonic , code is synced now, kindly requesting for your review, thank you!

File: tests/generic_config_updater/test_mgmt_interface.py [hunk 1]
```diff
@@ -37,11 +37,25 @@ def ensure_dut_readiness(duthost):
...
+                "forced_mgmt_routes"
+            ]
+        ) if duthost.is_multi_asic else create_path(
+            [
+                "MGMT_INTERFACE",
+                "eth0|{}".format(interface_address),
+                "forced_mgmt_routes"
+            ]
+        )
+    )
```
Comment (xincunli-sonic) [inline tests/generic_config_updater/test_mgmt_interface.py:55]: No need for wrapping the namespace, format_json_patch_for_multiasic will do wrap. Can you please check other test PRs if they are dup?

**Comment by xincunli-sonic** [comment]
This PR could be dup with https://github.com/sonic-net/sonic-mgmt/pull/14098, please double check. Thanks!

**Pull request closed without merging**